huaweicloud/services/acceptance/evs/data_source_huaweicloud_evs_volumes_test.go

```diff
@@ -102,5 +107,64 @@ data "huaweicloud_evs_volumes" "test" {
   server_id         = huaweicloud_compute_instance.test.id
   status            = "in-use"
 }
+
+# Filter using volume ID
+data "huaweicloud_evs_volumes" "volume_id_filter" {
+  volume_id = data.huaweicloud_evs_volumes.test.volumes.0.id
+}
+
+locals {
+  volume_id = data.huaweicloud_evs_volumes.test.volumes.0.id
+}
+
+output "is_volume_id_filter_useful" {
+  value = length(data.huaweicloud_evs_volumes.volume_id_filter.volumes) > 0 && alltrue(
+    [for v in data.huaweicloud_evs_volumes.volume_id_filter.volumes[*].id : v == local.volume_id]
+  )
+}
+
+# Filter using name
+data "huaweicloud_evs_volumes" "name_filter" {
+  name = data.huaweicloud_evs_volumes.test.volumes.0.name
+}
+
+locals {
+  name = data.huaweicloud_evs_volumes.test.volumes.0.name
```

Comment:
It seems that this local variable is not used.

Reply:
fixed.